### PR TITLE
Pr/icon expansion

### PR DESCRIPTION
**Pull request by MethodenMann**

### CODE REVIEW

File: packages/tocco-ui/package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
   },
   "devDependencies": {
     "@fortawesome/fontawesome-svg-core": "^1.2.8",
+    "@fortawesome/free-brands-svg-icons": "^5.5.0",
```
Comment (niklaushug) [inline packages/tocco-ui/package.json]: Though icons are chunked and delivered separately, it has an impact on loading time. As I added fortawesome I have decided not to include brands. Are brand icons requested now? How many brand icons are used? We still have the possibility to assemble an own set of brand icons.

File: packages/tocco-ui/src/Icon/Icon.js [hunk 2]
```diff
@@ -40,9 +40,12 @@ class Icon extends React.Component {
   }
 
   render() {
-    const filteredProps = _omit(this.props, ['dense', 'onLoaded', 'position', 'theme'])
```
Comment (niklaushug) [inline packages/tocco-ui/src/Icon/Icon.js]: Why was filteredProps removed? It ensures that only props are passed down to react-fontawesome which are consumed. All props which are only consumed by Icon are removed by _omit.

File: packages/tocco-ui/src/Icon/Icon.js [hunk 3]
```diff
@@ -40,9 +40,12 @@ class Icon extends React.Component {
...
+    const icon = (typeof this.props.icon === 'string' && this.props.icon.includes(','))
```
Comment (niklaushug) [inline packages/tocco-ui/src/Icon/Icon.js]: When prop.icon is defined correctly this code is not needed. May extend PropTypes validation by allowing only strings without comma and remove this code.

File: packages/tocco-ui/src/Icon/example.js [hunk 4]
```diff
@@ -9,8 +9,8 @@ export default () => {
   return (
     <div>
       {/* start example */}
-      <Icon icon="address-card" /><br />
-      <Icon icon={['far', 'address-card']} /><br />
+      <Icon icon="address-card" /><Icon icon="fab, microsoft"/><br />
```
Comment (niklaushug) [inline packages/tocco-ui/src/Icon/example.js]: Define as array and use curly braces like in the following line.

File: packages/tocco-ui/src/Icon/Icon.js (hunk 3 above)
Comment (MethodenMann) [reply packages/tocco-ui/src/Icon/Icon.js]: I dont quite get your comment. That you can use a comma separated string is a feature. I think in most cases it's friendlier this way.

File: packages/tocco-ui/src/Icon/Icon.js (hunk 2 above)
Comment (MethodenMann) [reply packages/tocco-ui/src/Icon/Icon.js]: Yes, i know. So what is there in "props" except the icon attribute which i handle separately?

File: packages/tocco-ui/src/Icon/example.js (hunk 4 above)
Comment (MethodenMann) [reply packages/tocco-ui/src/Icon/example.js]: As i said above this is something i explicitly want to support so i want to show it in the example

**Pull request merged**